.travis.yml

```diff
@@ -47,8 +49,7 @@ install:
     - export LINKFLAGS="${LDFLAGS}"
     - mkdir build
     - cd build
-    - cmake -DDOWNLOAD_GTEST=ON -DCMAKE_PREFIX_PATH=$CONDA_PREFIX ..;
+    - cmake -DBUILD_TESTS=ON ..
     - make -j2 test_xtensor_io
-    - cd test
 script:
-    - ./test_xtensor_io
+    - make xtest
```

Comment:
Same question here. Since you explicitly build the test_xtensor_io target, no need to "build" the xtest target.

Reply:
:+1: